taxonomies/labels.txt

```diff
@@ -18604,6 +18626,120 @@ wikidata:en:Q2028860
 label_categories:en: en:Fishing labels
 auth_url:en:https://www.asc-aqua.org
 
+en:FAO fishing areas, FAO major fishing areas, FAO inland waters
+bg:Главните Риболовни Зони ФАО
+cs:Hlavních Rybolovných Oblastí FAO
+da:Overordnede FAO-Fiskeriområder
+de:FAO-Fischereigebiete, FAO major fishing areas
+el:Μείζονες Αλιευτικές Ζώνες FAO
+es:Principales zonas pesqueras de la FAO, Principales áreas de pesca de la FAO
+et:FAO Peamiste Püügipiirkondade
+fi:FAO-Pääkalastusalueista
+fr:Principales zones de pêche de la FAO, Zone de pêche, Zone de pêche FAO
+ga:Mórlimistéir Iascaireachta FAO
+hr:ribolovno područje FAO, FAO, ribolovno područje
+hu:Fő FAO Halászati területek, FAO halászati körzetek
+it:Principali zone di pesca FAO
+lt:FAO Pagrindiniame Žvejybos Rajonu
+lv:FAO Galveno Zvejas Apgabalos
+mt:Taż-żoni Prinċipali Tas-sajd Tal-FAO
+nl:Grote visgebieden van de FAO, FAO major fishing areas
+pl:Głównych Obszarów Połowowych FAO, Obszary połowowe FAO
+pt:Zonas principais de pesca FAO
+ro:Principalele Zone de Pescuit FAO
+sk:FAO Hlavných Oblastí Rybolovu
+sl:Glavnih Ribolovnih Območij FAO
+sv:FAO:s Fiskeregioner
+xx:FAO major fishing areas
+description:en:European Union, FAO major fishing areas
```

Comment:
note that this is also addressed in the origins taxonomy

Reply:
Thanks for pointing it out

Reply:
I removed them from labels and added synonyms in origins instead.